README.md

```diff
@@ -2522,3 +2522,13 @@ typically want to run on your "performance" cores.
 ## Stargazers over Time
 
 [![Stargazers over Time](https://starchart.cc/mhx/dwarfs.svg?variant=adaptive)](https://starchart.cc/mhx/dwarfs)
+
+## Notable users
```

Comment:
Can you add a link from the `Table of contents` near the top of the README?

Reply:
Do we really needs two ToC?
![image](https://github.com/user-attachments/assets/fcadf978-e4de-49cd-ade4-157441ab9452)

Reply:
> Do we really needs two ToC?

I didn't even know that other ToC existed. :)

But: I also don't think this available in the Github app. And the ToC is truncated when viewing on mobile.

So yeah, I guess I'd like to keep the ToC in the README for now.